package.json

```diff
@@ -1,21 +1,24 @@
 {
   "name": "respec-caniuse-route",
-  "version": "2.0.1",
+  "version": "3.0.0",
   "description": "API endpoint handler for getting caniuse data in a ReSpec friendly way. Used in respec.org",
   "main": "index.js",
+  "type": "module",
   "keywords": [
     "respec",
     "caniuse"
   ],
   "repository": "git@github.com:sidvishnoi/respec-caniuse-route.git",
   "author": "Sid Vishnoi <sidvishnoi8@gmail.com>",
   "license": "MIT",
-  "bin": {
-    "fetch-caniuse-data": "./scraper/get-data.sh",
-    "convert-caniuse-data": "./scraper/process-data.js"
-  },
   "scripts": {
-    "fetch": "./scraper/get-data.sh",
-    "convert": "./scraper/process-data.js"
+    "build": "tsc -d",
+    "clean": "rm -rf *.js *.d.ts",
+    "dev": "tsc -w",
+    "prepare": "npm run clean && npm run build"
```

Comment:
npm will still have only the JS version (and `.d.ts` files). We don't need to migrate respec.org to TypeScript